src/testing/tests.rs

```diff
@@ -58,9 +59,10 @@ fn proper_init() {
     assert_eq!(
         config_response,
         ConfigResponse {
-            lottery_contract: HumanAddr::from(MOCK_HUB_CONTRACT_ADDR),
-            cw20_token_contract: HumanAddr::from(MOCK_CW20_CONTRACT_ADDR),
+            admin: HumanAddr::from(MOCK_HUB_CONTRACT_ADDR),
```

Comment:
Again why removed loterry contract?
The design I thought: when a round finishes lotterry contract sends jackpot tax reward to reward contract and then triggers update global index

Reply:
I would not make this admin

Reply:
If this is for multiple game contracts connecting to reward contract, then changing it to address is a good idea.